pkgs/pkgs-lib/tests/formats.nix

```
@@ -425,4 +425,48 @@ in runBuildTests {
       \u0627\u0644\u062c\u0628\u0631 = \u0623\u0643\u062b\u0631 \u0645\u0646 \u0645\u062c\u0631\u062f \u0623\u0631\u0642\u0627\u0645
     '';
   };
+
+  phpAtoms = shouldPass rec {
+    format = formats.php { finalVariable = "$config"; };
```

```suggestion
    format = formats.php { finalVariable = "config"; };
```

Though I don't see any test failure in CI 🤔

---

oops D: